setup.py

```diff
@@ -80,46 +103,33 @@
         "dominate",
     ],
     python_requires=">=3.5",
-    classifiers=[
```

Comment:
Moved these a bit higher up so that all the dependencies (install and extras) are closer together.